draft-ietf-trans-rfc6962-bis.md

```diff
@@ -985,8 +988,7 @@ The log stores information about its Merkle Tree in a `TreeHeadDataV2`:
 
 The length of NodeHash MUST match HASH_SIZE of the log.
 
-`timestamp` is the current NTP Time [RFC5905], measured in milliseconds since
-the epoch (January 1, 1970, 00:00 UTC), ignoring leap seconds.
+`timestamp` is the current, unsigned Unix Time [UNIXTIME].
```

Comment:
My proposed text:
"timestamp" is the current date and time, in the form of a 64-bit unsigned number of milliseconds elapsed since the Unix Epoch (1 January 1970 00:00:00 UTC - see [UNIXTIME]), ignoring leap seconds, in network byte order.

Reply:
Changed.